src/test/ui/macros/macros-nonfatal-errors.stderr

```diff
@@ -221,5 +215,14 @@ error: trace_macros! accepts only `true` or `false`
 LL |     trace_macros!(invalid);
    |     ^^^^^^^^^^^^^^^^^^^^^^
 
+error: cannot find macro `asm` in this scope
```

Comment:
This test needs to `use std::arch::asm;` so that it tests what it's intending to test.